examples/cdn/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wasmer JavaScript SDK</title>
+    <script defer type="module">
+        import { init, Wasmer } from "https://unpkg.com/@wasmer/sdk@latest/dist/Library.mjs";
```

Comment:
Doing this doesn't work? (I guess because this is getting the mjs instead of the js?).

```suggestion
        import { init, Wasmer } from "https://unpkg.com/@wasmer/sdk@latest";
```

(just double checking)

Reply:
Yeah. The `https://unpkg.com/@wasmer/sdk@latest` URL redirects to whatever file your `package.json`'s `unpkg` field points to, which is the UMD module in our case. We actually want to import the ES Modules version, so we need to use explicit paths.